Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -955,7 +881,7 @@ protected override NodeProperties CreatePropertiesObject()
 
         public override void Close()
```

Comment:
>Close [](start = 29, length = 5)

What's the relationship between `Close` and `Dispose`?  Is there a danger of leaving the object alive with a disposed `fileWatcher`?

Reply:
`Close` calls `Dispose`